### ISSUE TITLE
copyFiles to copy dir as is

### ISSUE THREAD

**Issue by danaki** [symfony/webpack-encore#570]
In my legacy project tinymce with jquery is used. It requires plugins and themes to be found in particular directories relative to tinymce.js. It loads them dynamically depending on config. However I can't find a combination of options for copyFiles() to copy whole module under build/ without altering or minfying. This is what I've tried, for example:
```
    .copyFiles([
        {from: 'node_modules/tinymce/plugins', to: 'tinymce', includeSubdirectories: true, pattern: /.*/},
    ])
```
It's just for plugins (but I need whole node_modules/tinymce/ untouched), and it minifies and merges all the plugins to build/tinymce file. Other options I've tried produce different errors sometimes confusing like:

```
Error: ENOENT: no such file or directory, open '/www/site/public/build/tinymce/'
verbose 27.15772796 Error: Command failed with exit code 1.
    at ProcessTermError.ExtendableBuiltin (/usr/local/Cellar/yarn/1.15.2/libexec/lib/cli.js:721:66)
    at ProcessTermError.MessageError (/usr/local/Cellar/yarn/1.15.2/libexec/lib/cli.js:750:123)
    at new ProcessTermError (/usr/local/Cellar/yarn/1.15.2/libexec/lib/cli.js:790:113)
    at /usr/local/Cellar/yarn/1.15.2/libexec/lib/cli.js:36900:30
    at Generator.throw (<anonymous>)
    at step (/usr/local/Cellar/yarn/1.15.2/libexec/lib/cli.js:304:30)
    at /usr/local/Cellar/yarn/1.15.2/libexec/lib/cli.js:317:13
    at processTicksAndRejections (internal/process/task_queues.js:86:5)
error Command failed with exit code 1
```

Is it possible to produce error output to help debugging?

**Comment by Lyrkan** [comment]
Hi @danaki,

You are missing some placeholders in the `to` option (see [this example](https://github.com/symfony/webpack-encore/blob/ba08baff7d9d90fe7e543bab021a90a0b57dffce/index.js#L491-L497)), try this instead:

```js
Encore.copyFiles([{
    from: './node_modules/tinymce/plugins',
    to: './tinymce/[path][name].[ext]',
    includeSubdirectories: true,
    pattern: /.*/
}]);
```

**Comment by danaki** [comment]
@Lyrkan Ok, it works with plugins. But if try to copy the whole directory like this:
```
    .copyFiles({
        from: './node_modules/tinymce/',
        to: './js/tinymce/[path][name].[ext]',
        includeSubdirectories: true,
        pattern: /.*/
    })
```

I'm getting errors like:
```
 warning  in ./node_modules/tinymce/package.json

Module parse failed: Unexpected token m in JSON at position 0 while parsing near 'module.exports = __w...'
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token m in JSON at position 0 while parsing near 'module.exports = __w...'
    at JSON.parse (<anonymous>)
    at parseJson (/www/site/node_modules/json-parse-better-errors/index.js:7:17)
    at JsonParser.parse (/www/site/node_modules/webpack/lib/JsonParser.js:16:16)
    at doBuild.err (/www/site/node_modules/webpack/lib/NormalModule.js:460:32)
    at runLoaders (/www/site/node_modules/webpack/lib/NormalModule.js:342:12)
    at /www/site/node_modules/loader-runner/lib/LoaderRunner.js:370:3
    at iterateNormalLoaders (/www/site/node_modules/loader-runner/lib/LoaderRunner.js:211:10)
    at iterateNormalLoaders (/www/site/node_modules/loader-runner/lib/LoaderRunner.js:218:10)
    at /www/site/node_modules/loader-runner/lib/LoaderRunner.js:233:3
    at runSyncOrAsync (/www/site/node_modules/loader-runner/lib/LoaderRunner.js:130:11)
    at iterateNormalLoaders (/www/site/node_modules/loader-runner/lib/LoaderRunner.js:229:2)
    at /www/site/node_modules/loader-runner/lib/LoaderRunner.js:202:4
    at process.nextTick (/www/site/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:73:15)
    at processTicksAndRejections (internal/process/task_queues.js:79:9)

 @ ./node_modules/tinymce sync ./node_modules/file-loader/dist/cjs.js?context=/www/site/node_modules/tinymce&name=./js/tinymce/[path][name].[ext] .*
 @ /private/var/folders/ss/xknqzv357r96bng6m1b47sfh0000gn/T/tmp-76300QwEIs2cI651k.tmp
```

Can I just copy the directory as is without attempting to modify it's contents?

**Comment by Lyrkan** [comment]
@danaki Which version of Encore are you using? There was an issue when using `Encore.copyFiles()` with JSON files, but it is supposed to be fixed since v0.25 (see https://github.com/symfony/webpack-encore/pull/536).

**Comment by danaki** [comment]
Ok, upgrading to last version helped.
